normalize_repos/normalize_repos.py

```diff
@@ -26,10 +29,16 @@ def get_cc_repos(github):
     cc = get_cc_organization(github)
     return cc.get_repos()
 
+def check_project_status(repo):
+    contents = repo.get_contents(".cc-metadata.yml")
+    contents = contents.decoded_content
+    metadata = yaml.load(contents, Loader=yaml.FullLoader)
+    return metadata["engineering_project"]
```

Comment:
Treat projects as non-engineering by default.
```suggestion
    return metadata.get("engineering_project", False)
```

Reply:
Done.